helper/users/dynamic/pool.go

```diff
@@ -81,6 +94,20 @@ func New(opts *PoolConfig) Pool {
 	}
 }
 
+// disable is an implementation of Pool that does not allow acquiring ugids
+type disable struct{}
```

Comment:
I would give it a more explicit name, that does not requiere reading the comments to know what it is

Reply:
good point ~ renamed to `noopPool` instead